colibris/app.py

```diff
@@ -103,11 +102,8 @@ def _init_routes(web_app):
 # APISpec/Swagger support
 
 def _init_swagger(web_app):
-    async def init_wrapper(app):
-        setup_swagger(app=app, swagger_url=settings.API_DOCS_PATH, swagger_info=app['swagger_dict'])
-
+    setup_swagger(app=web_app)
```

Comment:
@ccrisan, do you see a problem with this change? Why did we call `setup_swagger` on startup instead of right away?

Reply:
It might have been due to necessity to delay accessing settings. You might want to make sure settings.API_DOCS_PATH is properly initialized from project's settings at this point.